homework-3/src/main/java/com/epam/cashier/controller/service/repository/ProductRepository.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.cashier.controller.service.repository;
+
+import com.epam.cashier.controller.service.model.Product;
+
+import java.util.List;
+
+public interface ProductRepository {
+    List<Product> getAllProducts();
+
+    Product getProduct(int id);
```

Comment:
Total nitpick, but I would use getProductById, it has more readability.

Reply:
fixed